[RU] Add Beatmap Nominators (#4602)
ppy/osu-wiki: wiki/People/The_Team/Beatmap_Nominators/ru.md

---
tags:
  - BN
  - BNG
  - БН
  - Номинатор
---

# Номинаторы

> *См. также: [правила работы](Rules), [ожидания от номинаторов](Expectations), [общая информация](General_Information), [как стать номинатором](Becoming_a_Beatmap_Nominator)*

Reviewer: ```suggestion
*См. также: [правила для номинаторов](Rules), [ожидания от номинаторов](Expectations), [общая информация](General_Information), [как стать номинатором](Becoming_a_Beatmap_Nominator)*
```
Не согласен с «правилами работы», так как это звучит как что-то техническое, а там скорее по поводу поведения.

**Номинаторы** (также ***BN***, ***БН***) — группа пользователей, занимающихся [номинированием карт на ранк](/wiki/Beatmap_ranking_procedure). На форуме и в обсуждениях карт их имена имеют фиолетовый цвет.

## Роли и обязанности

> Номинаторы отвечают за [моддинг](/wiki/Modding) карт в категории [Pending](/wiki/Beatmaps#work-in-progress-and-pending) и их дальнейшее продвижение по процедуре ранкинга. К ним стоит обращаться за советом или просьбами, связанными с моддингом и маппингом. В возможности номинаторов входит [накладывание вето](Beatmap_Veto) на карты, которые не нарушают критерии ранкинга, но, по их мнению, недостаточно хорошо сделаны.

Reviewer: ```suggestion
Номинаторы отвечают за [моддинг](/wiki/Modding) карт в категории [Pending](/wiki/Beatmaps#work-in-progress-and-pending) и их дальнейшее продвижение по процедуре ранкинга. К ним стоит обращаться за советом или просьбами, связанными с моддингом и маппингом. В возможности номинаторов входит [накладывание вето](Beatmap_Veto) на карты, которые не нарушают критерии ранкинга, но, по их мнению, имеют проблемы, которые перед ранком нужно решить.
```
Поправил, чтобы было ближе к оригиналу, «недостаточно хорошо сделаны» далеко не всегда подходит к ситуациям, когда вето накладывают.

> Номинаторы отвечают за [моддинг](/wiki/Modding) карт в категории [Pending](/wiki/Beatmaps#work-in-progress-and-pending) и их дальнейшее продвижение по процедуре ранкинга. К ним стоит обращаться за советом или просьбами, связанными с моддингом и маппингом. В возможности номинаторов входит [накладывание вето](Beatmap_Veto) на карты, которые не нарушают критерии ранкинга, но, по их мнению, имеют проблемы, которые перед ранком нужно решить.

Reviewer 2: ```suggestion
Номинаторы отвечают за [моддинг](/wiki/Modding) карт в категории [Pending](/wiki/Beatmaps#ожидающие) и их дальнейшее продвижение по процедуре ранкинга. К ним стоит обращаться за советом или просьбами, связанными с моддингом и маппингом. В возможности номинаторов входит [накладывание вето](Beatmap_Veto) на карты, которые не нарушают критерии ранкинга, но, по их мнению, имеют проблемы, которые перед ранком нужно решить.
```

Author: will do for now

В дополнение к этому, номинаторы должны соответствовать [ожиданиям](Expectations) — набору рекомендаций по тому, как вести деятельность и помогать другим мапперам и сообществу в целом.

Номинаторы делятся на две группы: полноправные (Full Beatmap Nominators) и на испытательном сроке (Probationary Beatmap Nominators).

### Полноправные номинаторы

Помимо регулярного моддинга и ранкинга карт, полноправные номинаторы могут дисквалифицировать карты в категории Qualified, содержащие ошибки или недочёты. Они также могут участвовать в отборе заявок в номинаторы, если заинтересованы в продвижении в NAT ([команду оценки номинаций](/wiki/People/The_Team/Nomination_Assessment_Team)).

> Примерно раз в три месяца деятельность полноправных номинаторов проверяется NAT. Если кто-то из номинаторов ведёт себя неподобающе или номинирует карты, которые потом часто дисквалифицируют, его отправляют на испытательный срок. Этого не происходит с номинаторами, которые редко модят — вместо этого их со временем исключают из команды, если активность остаётся низкой.

Reviewer: ```suggestion
Примерно раз в три месяца деятельность полноправных номинаторов проверяется NAT. Если кто-то из номинаторов ведёт себя неподобающе или номинирует карты, которые потом часто дисквалифицируют, его отправляют на испытательный срок. Этого не происходит с номинаторами, которые редко модят — вместо этого их со временем исключают из команды, если их активность останется на низком уровне.
```

> 

Reviewer 3: this header has 3 paragraphs in the original, might be outdated?

Author: I merged the last two into one

### Номинаторы на испытательном сроке

Испытательный срок нужен, чтобы присмотреться к новым или «слабым» номинаторам. Их главная роль на этой позиции — помогать ранкать карты, при этом для каждого режима игры, имеющегося в мапсете, нужно одобрение как минимум одного полноправного номинатора.

> Длина испытательного срока — один месяц. Если номинатор справляется с требованиями и хорошо себя ведёт, его повышают до полноправного номинатора. В противном случае, ему продляют срок ещё на месяц, а затем исключают из команды, если он по-прежнему не выполняет требования.

Reviewer 4: ```suggestion
Длина испытательного срока — один месяц. Если номинатор справляется с возложенными на него обязанностями и не нарушает правил, его повышают до полноправного номинатора. В противном случае, испытательный срок продлевается ещё на один месяц. Если он по-прежнему не справляется со своими задачами, в конечном итоге его исключают из команды.
```

| ![][flag_US] [Nana Abe](https://osu.ppy.sh/users/6443117) |  |
| ![][flag_RU] [Shmiklak](https://osu.ppy.sh/users/5504231) | Русский |
| ![][flag_RU] [SMOKELIND](https://osu.ppy.sh/users/9327302) | Русский |
| ![][flag_US] [UberFazz](https://osu.ppy.sh/users/8646059) |  |

#### osu!taiko

| Имя | Языки |
| :-- | :-- |
| ![][flag_HK] [Cynplytholowazy](https://osu.ppy.sh/users/3901754) | Китайский, Кантонский диалект |
| ![][flag_EC] [Gamelan4](https://osu.ppy.sh/users/9856910) | Испанский |
| ![][flag_NL] [ikin5050](https://osu.ppy.sh/users/4007649) | Нидерландский |
| ![][flag_MY] [Jerry](https://osu.ppy.sh/users/605973) |  |

#### osu!catch

| Имя | Языки |
| :-- | :-- |
| ![][flag_HK] [autofanboy](https://osu.ppy.sh/users/636114) | Китайский, Кантонский диалект |
| ![][flag_RU] [Nelly](https://osu.ppy.sh/users/4741164) | Русский |

#### osu!mania

| Имя | Языки |
| :-- | :-- |
| ![][flag_ID] [Arzenvald](https://osu.ppy.sh/users/3027421) | Индонезийский |
| ![][flag_CN] [Shima Rin](https://osu.ppy.sh/users/6089608) | Китайский |
| ![][flag_US] [Unpredictable](https://osu.ppy.sh/users/7560872) |  |

## История

> - Изначально команда, ранкающая карты, называлась [Beatmap Appreciation Team](/wiki/Modding/Beatmap_Appreciation_Team) (команда ценителей карт).

Reviewer: ```suggestion
- Изначально команда, ранкающая карты, называлась [Beatmap Appreciation Team](/wiki/Modding/Beatmap_Appreciation_Team) (команда оценки карт).
```
Твой вариант звучит неофициально и как-то слишком по-любительски, как будто они просто любят карты.

Author: мне сложно сказать, что тут имелось в виду. почему-то мне кажется, что к названию приложил руку либо Эфемерал, либо кто-то на него похожий, уж очень это отдаёт духом triumvir conglomerate, LSD и прочими вычурными/нестандартными формулировками. возьму твой вариант, потому что так глубоко вряд ли кто-нибудь в принципе будет думать

> - 22 августа 2014 года BAT реформировали, лишив их прав на модерирование и красного ника на форуме и в чате.

Reviewer: ```suggestion
- 22 августа 2014 года BAT реформировали, лишив их красного ника на форуме и в чате и прав на модерирование.
```
Поменял бы порядок, так как при первом прочтении кажется, что их лишили «прав на красного ника», а не просто красного ника. Только мне близость двух «и» не нравится, но ты что-нибудь придумаешь!

Author: а мне нравится
